### PR TITLE
feat: load rmrk2 image on gallery item

### LINKED ISSUE #5183: Load image rmrk2 that doesn't have `mediaUri`
@kodadot/internal-dev does anyone knows how to load an image for rmrk2-nft that doesn't have `mediaUri` in their metadata?

- example: https://beta.kodadot.xyz/rmrk2/gallery/16882818-3289d9f38266576b1c-AMERICAS2_S01EPISODES-EPISODE_17_SE-00000001?page=1
- metadata: https://image-beta.w.kodadot.xyz/ipfs/QmPQAj1qrBBBJaYHCYvF9WHa3Y7SnFwEG9JtNB7XYApk8x/Episode_17_Alaska_SE_Meta.json

```json
{
  "description": "To be totally honest, Alaska was the state we were looking forward to the most and sadly is the state that disappoints us the most. Instead of the planned four weeks we only stay for two and in these two weeks we mostly sit in the car and look out into pouring rain, before we decide we've had enough and flee. \n\nThe Alaskan scenery is majestic, but the summer months also mean a ton of tourists and most mornings we don't wake up alone in a serene mountain world as imagined, but rather right next to a well-paved highway with trucks roaring past every other minute. Or on the lit-up parking lot of a local outdoor shop that allows travelers to spend the night. We are very sure Alaska can hold up to every dreamy imagination, it just wasn't the right timing for us.\n\n-----------\n\nNote: Limited Edition and Directors Cut episodes have been merged and are now called LE/DC so that you get the pictures from the DC for the price of the LE!\n\nA free, personal non-commercial use license for all media included in this NFT is granted to it's owner.(CC-NC-BY-SA)\n\nThis NFT is part of the [AMERICAS 2.0](https://twitter.com/Americas_2_0) project. Collect the limited edition/directors cut [LE/DC] episodes in the [COLLECTORS MAP](https://singular.app/collectibles/kusama/3289d9f38266576b1c-AMERICAS2_S01MAP). Collect the supporters edition [SE] in the [SUPPORTERS MAP](https://singular.app/collectibles/kusama/3289d9f38266576b1c-AMERICAS2_S01MAP). If you complete a season, and on the road to completion, different perks await - depending on the edition you collect. For details check the MAP's description.",
  "name": "S01E17 - Alaska [SE]"
}
```

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5183

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

| &nbsp; | &nbsp; |
| --- | --- |
| Screenshot 2023-03-15 at 23 25 15 | Screenshot 2023-03-15 at 23 25 33 |

### CODE REVIEW

**Review by daiagi — APPROVED** [review]

File: components/base/types.ts [hunk 1]
```diff
@@ -14,7 +16,7 @@ export type BaseTokenType<T = BaseMintedCollection> = {
   secondFile: File | null
 }
 
-export type CarouselNFT = {
+export interface CarouselNFT extends RMRK2Resources {
```
Comment (vikiival) [inline components/base/types.ts:19]: RMRK2Resources
```suggestion
export interface CarouselNFT extends ItemResources {
```

File: composables/useNft.ts [hunk 2]
```diff
@@ -2,10 +2,11 @@ import type { NFT, NFTMetadata } from '@/components/rmrk/service/scheme'
 import { sanitizeIpfsUrl } from '@/utils/ipfs'
 import { processSingleMetadata } from '@/utils/cachingStrategy'
 
-type RMRK2Resources = {
-  resources: {
-    src: string
-    thumb: string
+export type RMRK2Resources = {
```
Comment (vikiival) [inline composables/useNft.ts:5]:
```suggestion
export type ItemResource = {
```

**Review by vikiival — CHANGES_REQUESTED** [review]
I want to avoid naming anything RMRK

**Comment by vikiival** [comment]
@preschian can we please proceed? 🥺

**Pull request merged**